### DISCUSSION TITLE
Are Bosch A/C units supported?

### DISCUSSION THREAD

**Discussion by tiimjcb** [thoukydides/homebridge-homeconnect#92] [Q&A]
greetings everyone.

i'm looking to buy an A/C for my room, but i want it to be homebridge compatible, or homekit controlled.
i've seen multiple A/C, especially one of the Climate Class 8000i. it says it's controlled by the HomeCom app (and not Home Connect, weird). it would be my first Bosch product, so that's why i'm asking.

is this A/C supported by the plugin?

thanks for anyone who answers.
have a great day.

**Comment by thoukydides** [comment]
Sorry, no.

This plugin only supports Home Connect. As you can see in the [README](https://github.com/thoukydides/homebridge-homeconnect#readme) this is mainly limited to kitchen appliances:
* Cooking Appliances
  * CoffeeMaker
  * CookProcessor
  * Hob
  * Hood
  * Oven
  * WarmingDrawer
* Cleaning Appliances
  * CleaningRobot
  * Dishwasher
  * Dryer / Washer / WasherDryer
* Cooling Appliances
  * Freezer / FridgeFreezer / Refrigerator
  * WineCooler

Home Connect does not support any A/C devices, despite any similarity in name.

**Reply by tiimjcb** [reply]
okay, thanks for the answer. so maybe i should look at another plugin, or just don't choose thag A/C.

thanks for answering again, have a great day!

**Comment by thoukydides** [comment]
The Home Connect API added AirConditioner support on 5th August 2025. This is supported in [v1.6.0](https://github.com/thoukydides/homebridge-homeconnect/releases/tag/v1.6.0) of this plugin.
